### PR TITLE
Add sources and search guide pages to the new header

### LINKED ISSUE #1987: Add Sources and Search Guide pages to the new header
## Problem

The new header/footer navbar is missing the Sources and Search guide pages. The pages are already available in the `/pages` directory, they just need to be added to the `use-pages` composable.

## Description

We need to add the pages here, between the Licenses and Get involved pages:
https://github.com/WordPress/openverse-frontend/blob/cc3f802c2d727417bfb2adb9f268e8739f9a1723/src/composables/use-pages.ts#L13-L22

```
        {
          id: 'sources',
          name: 'navigation.sources',
          link: app.localePath('/sources'),
        },
        {
          id: 'search-help',
          name: 'header.search-guide-nav-item',
          link: app.localePath('/search-help'),
        },
```

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by AlexVCS**
## Fixes
Fixes #1987 by @obulat

## Description
Added source and search guide pages to the new header

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
       ⬆️ Currently I'm getting an error, but I don't think it relates to any of the changes I made.

Screen Shot 2022-11-18 at 1 27 20 PM

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/composables/use-pages.ts
Comment (obulat) [inline src/composables/use-pages.ts:25]:
```suggestion
          name: 'navigation.search-help',
```

Comment (obulat) [reply src/composables/use-pages.ts:25]: The issue was incorrect about this.

**Review by obulat — COMMENTED** [review]
Thank you for your PR, @AlexVCS!

The repository recently changed from using husky to pre-commit. This is probably causing the error you are getting. To fix it, run Undo Husky config `git config --unset core.hooksPath`

**Comment by AlexVCS** [comment]
> Thank you for your PR, @AlexVCS!
> 
> The repository recently changed from using husky to pre-commit. This is probably causing the error you are getting. To fix it, run `Undo Husky config git config --unset core.hooksPath`

Thanks! I've tried writing that command different ways and I keep getting command not found. I've run ```volta install pnpm
``` again to make sure all the dependencies were installed but it's not working. I'll push up the change you recommended but if you've got a different command that could stop that error please let me know!

**Review by obulat — COMMENTED** [review]
Would you be comfortable running the Playwright tests in Docker to update the snapshots for failing tests, @AlexVCS? It's OK if not, I can add the updates if you need.

**Comment by AlexVCS** [comment]
> Would you be comfortable running the Playwright tests in Docker to update the snapshots for failing tests, @AlexVCS? It's OK if not, I can add the updates if you need.

I'll try and do it, thanks!

**Comment by AlexVCS** [comment]
I ran the Playwright tests and the error I'm getting throughout is this
``` Error: Cannot find module '~/locales/en.json'```

I know that with #1949 the file was changed to en.json5 and I'm guessing that's why this error is happening?

**Comment by obulat** [comment]
> I ran the Playwright tests and the error I'm getting throughout is this
> ``` Error: Cannot find module '~/locales/en.json'```
> 
> I know that with #1949 the file was changed to en.json5 and I'm guessing that's why this error is happening?

Can you run `pnpm dev`? That should also run the necessary i18n scripts

**Comment by AlexVCS** [comment]
> > I ran the Playwright tests and the error I'm getting throughout is this
> > ` Error: Cannot find module '~/locales/en.json'`
> > I know that with #1949 the file was changed to en.json5 and I'm guessing that's why this error is happening?
> 
> Can you run `pnpm dev`? That should also run the necessary i18n scripts

the same error I was getting initially persists

**Comment by obulat** [comment]
Hi @AlexVCS, I rebased your branch onto `main`. The problems with localse should be fixed. Could you try to run `pnpm dev` on updated branch? Please, let me know if you have any problems.

**Comment by obulat** [comment]
I've added some snapshots to your PR, @AlexVCS. Judging by the snapshots, we will need to fix https://github.com/WordPress/openverse-frontend/issues/2012 before we can merge this PR to make sure that the footer doesn't break the layout.

**Comment by zackkrida** [comment]
Blocked until #2012 is closed.

**Comment by dhruvkb** [comment]
#2012 has been merged so this is unblocked. @AlexVCS there are some merge conflicts, could you update the PR to resolve the conflicts?

**Comment by krysal** [comment]
@WordPress/openverse-frontend anyone interested in taking over this PR and finishing it?

**Comment by obulat** [comment]
I've re-based this PR onto main now that the internal header PR is merged. It is ready to be reviewed.

**Review by obulat — APPROVED** [review]

**Comment by AlexVCS** [comment]
sorry for my delays in responding. I hope to pick up other PR's for the project in the future

**Comment by obulat** [comment]
> sorry for my delays in responding. I hope to pick up other PR's for the project in the future

Welcome back, @AlexVCS! I've noticed that your fork of `openverse-frontend` had a stray lock file that was blocking you from updating it. Let me know if you need help getting it fixed.

**Review by dhruvkb — APPROVED** [review]
LGTM!

**Comment by AlexVCS** [comment]
> 

Thanks! It looks like I was able to update my fork and pull down those changes! You no longer see that stray lock file, do you?

**Comment by obulat** [comment]
> Thanks! It looks like I was able to update my fork and pull down those changes! You no longer see that stray lock file, do you?

Great! Yes, I see that your fork is updated and without any stray files.
Thank you for your contribution in this PR! 🎉

**Pull request merged**
